.github/workflows/test.yml

```diff
@@ -31,3 +31,6 @@ jobs:
 
     - name: Run tests
       run: lein test
+
+    - name: Checking examples
```

Comment:
you can also run cljstyle this way by using  github workflow similar to [this one](https://github.com/Flexiana/Frankie/blob/51451721bec3041cf5adf8cb08e1b4bceaf91b75/.github/workflows/check-format.yml). then you dont need to set up aliases in `project.clj`

Reply:
Thanks @petervlk, I'm using the shell script from command line for smoke-testing too.

Reply:
in that case, wouldn't it be better to put this into a separate workflow so that it is easier to see that these things are not complements but alternatives? just a thought :)

Reply:
I've created a new job for it